src/server.js

```diff
@@ -13,9 +17,34 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
 
+
 // Tell express to use your routers here
 const customerRouter = require('./routers/customer');
 app.use('/customers', customerRouter);
+const moviesRouter = require('./routers/movies');
+app.use('/movies', moviesRouter);
+const screensRouter = require('./routers/screens');
+app.use('/screens', screensRouter);
+
+// Error handling
+app.use((e, req, res, next) => {
+    if (e instanceof Prisma.PrismaClientKnownRequestError) {
+        if (e.code === "P2002") {
+            return res.status(409).json({ error: "A customer with the provided email already exists" })
+        }
+        if (e.code === "P2000") {
+            return res.status(409).json({ error: "Value provided was too long" })
+        }
+    }
```

Comment:
Not necessary, but this kind of situation is one of those rarer cases where a switch statement might come in handy. I don't like to use them very often, but sometimes they can be useful:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

```js
if (e instanceof Prisma.PrismaClientKnownRequestError) {
  switch (e.code) {
    case "P2002":
      return res.status(409).json({ error: "A customer with the provided email already exists" })
      break
    
    case "P2013"
      ...
  }
}
```